packages/config/config/devices/0x0438/touch-thermostat-16a.json

```diff
@@ -0,0 +1,348 @@
+			"defaultValue": 45
+		},
+
+		{
+			"#": "7",
+			"label": "Temperature Hysteresis",
+			"description": "Temperature control hysteresis. Allowable range: 5-20",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 20,
+			"unit": "0.1 °C",
+			"defaultValue": 5
+		},
+
+		{
+			"#": "8",
+			"label": "Energy Save Mode",
+			"description": "0 - Off, user schedule used. 1 - On, use Energy_Save_Heating temperature set by THERMOSTAT_SETPOINT_SET",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Energy save mode off",
+					"value": 0
+				},
+				{
+					"label": "Energy save mode on",
+					"value": 1
+				}
+			]
+		},
+
+		{
+			"#": "9",
+			"label": "Floor Sensor Type",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 4,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "NTC/10K",
+					"value": 1
+				},
+				{
+					"label": "NTC/15K",
+					"value": 2
+				},
+				{
+					"label": "NTC/50K",
+					"value": 3
+				},
+				{
+					"label": "NTC/100K",
+					"value": 4
+				}
+			]
+		},
+
+		{
+			"#": "10",
+			"label": "Temperature Control Reference",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Room sensor",
+					"value": 1
+				},
+				{
+					"label": "Floor sensor",
+					"value": 2
+				},
+				{
+					"label": "Room + Floor sensor",
+					"value": 3
+				}
+			]
+		},
+
+		{
+			"#": "11",
+			"label": "Room Temperature Correction",
+			"description": "Correct room temperature sensor. Allowable range: -10 - 10",
+			"valueSize": 1,
+			"minValue": -10,
+			"maxValue": 10,
+			"unit": "0.5 °C",
+			"defaultValue": 0
+		},
+
+		{
+			"#": "12",
+			"label": "Floor Temperature Correction",
+			"description": "Correct floor temperature sensor. Allowable range: -10 - 10",
+			"valueSize": 1,
+			"minValue": -10,
+			"maxValue": 10,
+			"unit": "0.5 °C",
+			"defaultValue": 0
+		},
+
+		{
+			"#": "13",
+			"label": "Drying Mode Duration",
+			"description": "Allowable range: 5-100",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 100,
+			"unit": "minutes",
+			"defaultValue": 30
+		},
+
+		{
+			"#": "14",
+			"label": "Mode After Drying",
+			"description": "Set the mode after drying mode completes.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off mode",
+					"value": 0
+				},
+				{
+					"label": "Manual mode",
+					"value": 1
+				},
+				{
+					"label": "Energy save mode",
+					"value": 2
+				},
+				{
+					"label": "Away mode",
+					"value": 3
+				}
+			]
+		},
+
+		{
+			"#": "16",
+			"label": "Temperature Report on Value",
+			"description": "When to report temperature relative change actively. Allowable range: 0 (disable), 2-10",
```

Comment:
Add a single option for disable instead of mentioning it specifically in the description